src/app/services/articles-service/index.ts

```diff
@@ -7,16 +7,15 @@ import {
   topicValues,
   sortValues,
   timeValues,
-} from '@modules/MainArticlesScreen/namespace/index';
-
-const now = dayjs();
+} from '@pages/MainArticlesScreen/namespace/index';
 
 const getArticlesService = (
   page = 1,
   topic?: string,
   sortBy?: string,
   date?: string,
 ) => {
+  const now = dayjs();
```

Comment:
if `now` is not reused anywhere, can we simply keep `const to dayjs().format(...)`?